presto-main/src/main/java/io/prestosql/sql/planner/sanity/ValidateDependenciesChecker.java

```diff
@@ -317,6 +317,7 @@ public Void visitOffset(OffsetNode node, Set<Symbol> boundSymbols)
         @Override
         public Void visitLimit(LimitNode node, Set<Symbol> boundSymbols)
         {
+            checkState(!node.isWithTies(), "Unexpected node: LimitNode with ties.");
```

Comment:
Why?

Reply:
This check is already removed in the following commit where final implementation of LimitNode with ties is added.

Reply:
Got it. I missed that when reviewing commits individually.